assets/src/edit-story/components/inspector/prepublish/checklistTab.js

```diff
@@ -0,0 +1,284 @@
+  color: ${({ theme }) => theme.colors.fg.secondary};
+  font-size: ${({ theme }) => theme.fonts.body1.size};
+  line-height: ${({ theme }) => theme.fonts.body1.lineHeight};
+  margin: 0;
+`;
+
+const EmptyParagraph = styled.p`
+  color: ${({ theme }) => theme.colors.fg.secondary};
+  font-size: ${({ theme }) => theme.fonts.description.size};
+  line-height: ${({ theme }) => theme.fonts.description.lineHeight};
+  margin: 0;
+`;
+
+function annotateNumber(number) {
+  if (number <= MAX_NUMBER_FOR_BADGE) {
+    return number;
+  }
+  return `${MAX_NUMBER_FOR_BADGE}+`;
+}
+
+const ChecklistTab = (props) => {
+  const { checklist } = props;
+
+  const { highPriority, recommended, pages } = useMemo(
+    () =>
+      checklist.reduce(
+        (prevMessages, current) => {
+          const id = uuidv4();
+          const currentMessage = { ...current, id };
+          const isHighPriority =
+            currentMessage.type === PRE_PUBLISH_MESSAGE_TYPES.ERROR;
+
+          const [typeKey, copyKey] = isHighPriority
+            ? ['highPriority', 'recommended']
+            : ['recommended', 'highPriority'];
+
+          if (currentMessage.page !== undefined) {
+            const prevTypeGroup = prevMessages.pages[typeKey] || {};
+            const prevPageGroup = prevTypeGroup[currentMessage.page] || [];
+
+            let prevLengths = { highPriority: 0, recommended: 0 };
+            if (prevMessages.pages?.lengths) {
+              ({ lengths: prevLengths } = prevMessages.pages);
+            }
+
+            return {
+              recommended: prevMessages.recommended,
+              highPriority: prevMessages.highPriority,
+              pages: {
+                ...prevMessages.pages,
+                [typeKey]: {
+                  ...prevTypeGroup,
+                  [currentMessage.page]: [...prevPageGroup, currentMessage],
+                },
+                lengths: {
+                  ...prevLengths,
+                  [typeKey]: prevLengths[typeKey] + 1,
+                },
+              },
+            };
+          }
+
+          const groupedMessages = {
+            [copyKey]: prevMessages[copyKey],
+            [typeKey]: [...prevMessages[typeKey], currentMessage],
+          };
+          return { pages: prevMessages.pages, ...groupedMessages };
+        },
+        {
+          highPriority: [],
+          recommended: [],
+          pages: {},
+        }
+      ),
+    [checklist]
+  );
+
+  const renderRow = useCallback(({ message, help, id, pageGroup }) => {
+    return (
+      <Row key={`guidance-${id}`} pageGroup={pageGroup}>
+        {message}
+        <HelperText>{help}</HelperText>
+      </Row>
+    );
+  }, []);
+
+  const renderPageGroupedRow = useCallback(
+    (entry) => {
+      const [pageNum, messages] = entry;
+      return (
+        <Fragment key={`guidance-page-group-${pageNum}`}>
+          <PageIndicator>
+            <Rectangle />
+            {sprintf(
+              /* translators: %s: page number */
```

Comment:
```suggestion
              /* translators: %s: page number. */
```

This is so small, but I've noticed an attempt and getting all translator strings to end in a period.